### ISSUE TITLE
[Feature request]: alerting 

### ISSUE THREAD

**Issue by mnuic** [grafana/grafana#8452]
It would be nice to have a possibilty to suppress alerts for a given time period. Let say I set automatic restart servers/aplications every day at 6:00 in the morning. I would like to suppress alerts from 5:45-6:15 because it is a regular thing and I don't want alerts to pop-out.

Second, grafana triggers an alert on a first occurrance, when it should be possible to define, let say 3 checks in 2 minutes period  and only after then it shoud send alert.

Thanks :)

**Comment by torkelo** [comment]
duplicate of https://github.com/grafana/grafana/issues/5856
